stage1_fly/run/main.go

```diff
@@ -113,7 +112,7 @@ func evaluateMounts(rfs string, app string, p *stage1commontypes.Pod) ([]flyMoun
 			return nil, fmt.Errorf("duplicate mount given: %q", m.Volume)
 		}
 		namedVolumeMounts[m.Volume] = volumeMountTuple{M: m}
-		log.Printf("Adding %+v", namedVolumeMounts[m.Volume])
+		stderr.Printf("Adding %+v", namedVolumeMounts[m.Volume])
```

Comment:
Where stderr defined?

Also, there are lots of calls to `fmt.Fprintf`. Do you plan to replace them with rktlog?